src/components/ContentNode/Caption.vue

```diff
@@ -31,6 +31,11 @@ export default {
       type: Boolean,
       default: false,
     },
+    tag: {
+      type: String,
+      required: false,
+      default: () => 'caption',
```

Comment:
We should add a validator to make clear that this can only ever be one of `'caption'` or `'figcaption'` and not any arbitrary HTML tag.

Reply:
Done https://github.com/apple/swift-docc-render/pull/692/commits/d814cd80923aba1bde7e919e7817c6c9d70b1415